.ci/jenkins/scripts/kind.groovy

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+kindVersion = env.KIND_VERSION ?: 'v0.20.0'
+kindRegistryName = "kind-registry"
+kindRegistryPort = "5001"
+
+void start(boolean debug = false) {
```

Comment:
Did you forget to load this script inside the helper.groovy to make the `kind` command context available? I don't see where this would happen otherwise (compare with what was for minikube).

Reply:
yup, also missing installation of the kind on our nodes, will add